bin/desi_run_night

```diff
@@ -74,6 +74,11 @@ def parse_args():  # options=None):
                              "are pruned using PROCCAMWORD to only process the remaining cameras not found to exist.")
     parser.add_argument("--use-tilenight", action="store_true",
                         help="Use desi_proc_tilenight for prestdstar, stdstar, and poststdstar steps.")
+    parser.add_argument("--only-completed-tiles", action="store_true",
+                        help="Set to restrict to completed tiles as defined by the table pointed to by specstatus_path.")
```

Comment:
In the spirit of `desi_run_night` being primarily for production runs and wanting default options to be the most likely "do the right thing", let's switch the sense of this option, i.e. make the default the equivalent of only-completed-tiles, and make the option something like `--all-tiles`.

Reply:
Resolved in 652e9dc